in_toto/models/layout.py

```diff
@@ -248,6 +282,38 @@ def __init__(self, **kwargs):
 
     self.validate()
 
+  def display(self):
+    """
+    <Purpose>
+      Represents a Step by returning a string which contains information pertaining to its member fields
+    <Returns>
+      Step Class fields as strings, 
+      i.e. name (threshold), 
+            pubkeys 
+            expected_command, 
+            expected_materials, 
+            expected_products
+    """
+    pubkeys = "{}".format("\n  ".join(self.pubkeys))
+    expected_command = "{}".format(" ".join(self.expected_command))
+    expected_materials = ""
+    expected_products = ""
+
+    for value in self.expected_materials:
+      expected_materials += " {}\n\t".format(" ".join(value))
```

Comment:
In my last review suggested to write a helper function to display artifact rules.

Reply:
Ooh, I missed this somehow... but a private method (`_display_list`) now takes care of it in the latest changes!